core/src/com/unciv/logic/automation/NextTurnAutomation.kt

```diff
@@ -25,8 +26,9 @@ class NextTurnAutomation{
         respondToTradeRequests(civInfo)
 
         if(civInfo.isMajorCiv()) {
+            offerDeclarationOfFriendship(civInfo)
             offerPeaceTreaty(civInfo)
-            exchangeTechs(civInfo)
```

Comment:
Why delete exchangeTechs? I think we could just move it below offerResearch

Reply:
exchangeTechs() has used the trade tech method, if we use exchangeTechs(), trade tech should not be removed